packages/web/src/components/NavBar/NavProfileMenu.tsx

```diff
@@ -10,28 +10,45 @@ import {
   Avatar,
 } from '@chakra-ui/react';
 
-export const NavProfileMenu: React.FC = () => (
-  <Menu variant="navbar">
-    <MenuButton as={Button}>
-      <Avatar size={'sm'} />
-    </MenuButton>
-    <MenuList>
-      <MenuItem>
-        <Link variant="navbarMenuItem" href="/app/profile">
-          Edit Profile
-        </Link>
-      </MenuItem>
-      <MenuItem>
-        <Link variant="navbarMenuItem" href="/app/contributions">
-          View Contributions
-        </Link>
-      </MenuItem>
-      <MenuDivider />
-      <MenuItem>
-        <Link variant="navbarMenuItem" href="/api/auth/github/logout">
-          Log Out
-        </Link>
-      </MenuItem>
-    </MenuList>
-  </Menu>
-);
+export const NavProfileMenu: React.FC = () => {
+  const [currentUserId, setCurrentUserId] = React.useState<string>('');
+
+  React.useEffect(() => {
+    const fetchCurrentUserID = async () => {
+      const res = await fetch(`/api/users/me`);
+
+      if (res.status == 200) {
+        const data = await res.json();
+        setCurrentUserId(data.id);
+      }
+    };
+
+    fetchCurrentUserID();
+  }, []);
+
+  return (
+    <Menu variant="navbar">
+      <MenuButton as={Button}>
+        <Avatar size={'sm'} />
+      </MenuButton>
+      <MenuList>
+        <MenuItem>
+          <Link variant="navbarMenuItem" href={`/user/${String(currentUserId)}`}>
+            View Profile
+          </Link>
+        </MenuItem>
```

Comment:
Make this a ternary on the truthiness of `currentUserId` and return `undefined` if it's not set

Reply:
In [this commit I added](https://github.com/AmericanAirlines/Project-X/pull/62/commits/a6da82c384b5642fb271d26dd81d5bc07cdcfb87), I instead put everything currently in the drop down menu in the ternary check for `currentUserId` and added a "log in" option instead of undefined if the user isn't logged in since it doesn't make sense to log the user out if they're not already logged in and they can't view their contributions if their not logged in.

Would this work / does it make sense?